messaging/callbacks/msgDelivered.md

```diff
@@ -25,7 +25,9 @@ MMS and Group messages <b>don’t</b> currently support delivery receipts. Howev
 | message.owner         | `string` | The phone number this particular message is associated with.<br> For an outbound message, this is always the `from` number.<br> For an inbound message, this will be (one of) the `to` number(s).<br>For instance, if this is an inbound group message, the `owner` field will be set to the `to` number that this particular copy of the group message belongs to. |
 | message.time          | `string` | The time stamp of when message was created                                                                                                                                                                                                                                                                                                                          |
 | message.direction     | `string` | Whether the message was sent from Bandwidth, or received by a Bandwidth number                                                                                                                                                                                                                                                                                      |
-| message.to            | `array`  | The phone number (or numbers) the message the message is sent to. On a POST, this can be a String, or an array of one or more numbers. In all other places, this will be an array.                                                                                                                                                                                  |
+| message.to            | `array`  | The phone number (or numbers) the message the message was sent to. This will always be formatted as an array in the callback.
+
+Note: On a POST request to create a message, this can be a String comprising a single number, or an array of one or more numbers. Bandwidth will always return the message.to value as an array in callbacks.                                                                                                                                                                         |
```

Comment:
This won't work how  you think it will, you'll need to put `<br>` where you want new lines within a table cell instead of a new line